README.md

```diff
@@ -119,7 +124,20 @@ By default, the binary is built as well. If you don't want/need it, then build l
 Or put in your `Cargo.toml` file:
 
 ```toml
-pulldown-cmark = { version = "0.4", default-features = false }
+pulldown-cmark = { version = "0.5", default-features = false }
+```
+
+SIMD accelerated scanners are available for the x64 platform from version 0.5 onwards. To
+enable them, build with simd feature:
+
+```bash
+> cargo build --features simd
+```
+
+Or add the feature to your project's `Cargo.toml`:
+
+```toml
+pulldown-cmark = { version = "0.5", default-features = false, features = ["simd"] }
```

Comment:
I wonder if we can remove the getopts default feature from library builds while leaving them in for binaries (the current state was added in #56), but I don't see an obvious way to do it, at least without having two separate crates, one for the lib and one for the bin. This is probably a pretty minor nit and shouldn't block this release.

Reply:
Hmmm yes, that would be nice. I don't see how we could do it either though…